### ISSUE TITLE
Warning about doing big updates every time

### ISSUE THREAD

**Issue by MrGoraj** [Jays2Kings/tachiyomiJ2K#1581]
### Describe your suggested feature

It feels like it's not a bug, which is why I'm reporting it as a "feature", but the app has for several versions now introduced a warning about doing a big library update every time you do it.

I haven't found an option anywhere to turn it off, and such information should appear only once for informational purposes, and when I confirm that I have read it, each subsequent display only annoys the user.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an request for a feature in the original Tachiyomi, I am opening because multiple versions of TachiyomiJ2K have been released without this feature.
- [X] If this is an issue with an extension, or a request for an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[1.6.6](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

**Comment by Jays2Kings** [comment]
It should continue to annoy the user. Reading it is pointless if you aren't taking action to solve the warning. It's like if your phone says "low battery please charge" at 20% and you ignore it when it shows up again at 15%, 10%, 5% and wonder why your phone is dead. In this case it would be a source either dying or asking to be removed from tachi's extension list.
